WordPress/Classes/ViewRelated/Blog/Site Settings/SiteSettingsViewController+Swift.swift

```diff
@@ -15,6 +16,32 @@ import WordPressFlux
 }
 
 extension SiteSettingsViewController {
+    // MARK: - General
+
+    @objc func showPrivacySelector() {
+        struct SiteSettingsPrivacyPicker: View {
+            let blog: Blog
+            @State var selection: SiteVisibility
```

Comment:
It's a bit awkward to use any SwiftUI view that accepts `Binding` from `UIKit` because just binding itself doesn't update the view.